extensions/smallrye-jwt/runtime/src/main/java/io/quarkus/smallrye/jwt/runtime/auth/JwtRolesMapper.java

```diff
@@ -0,0 +1,9 @@
+package io.quarkus.smallrye.jwt.runtime.auth;
```

Comment:
Same question, why is this intreface needed ?

Reply:
@andreas-eberle If there is a real case where the roles info is spread across multiple claims and when the existing smallrye properties can't really help the auto extraction of these roles, then lets consider this interface (that should live then in smallrye-jwt I guess) but having it just in case would be redundant. If you have some specific requirement/mapping case in mind then lets review, thanks

Reply:
The interface is there to allow for easy injection of own implementations of this roles mapping. Please also see my comment to your previous remark. I'm happy to move this interface to smallrye-swt. Should I prepare a PR?